src/platforms/browser/ImageWorker.mjs

```diff
@@ -144,7 +144,7 @@ const createWorker = function() {
     }
 
     ImageWorkerServer.isPathAbsolute = function(path) {
-        return /^(?:\/|[a-z]+:\/\/)/.test(path);
+        return /^(?:\/|[a-z]+:\/\/)/.test(path) || path.substr(0, 5) == 'data:';
```

Comment:
any reason to not test for strict equality?

Reply:
Using loose comparison operator for strings is a historical convention rather than anything else, but there are some obscure examples why you would actually want to avoid strict comparison between string types ([here](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons))

```javascript
"abc" == new String("abc")    // true
"abc" === new String("abc")   // false
```